tibanna/cw_utils.py

```diff
@@ -79,8 +79,8 @@ def get_metrics(self, nTimeChunks=1):
             max_disk_space_used_GB_chunks.append(self.max_disk_space_used())
             max_ebs_read_chunks.append(self.max_ebs_read())
         self.max_mem_used_MB = self.choose_max(max_mem_used_MB_chunks)
-        self.min_mem_available_MB = self.choose_min(min_mem_available_MB_chunks)
-        if self.max_mem_used_MB:
+        self.min_mem_available_MB = self.get_min(min_mem_available_MB_chunks)
```

Comment:
I am replacing `choose_min` with `get_min` here since we want to keep  `0.0` in the list. This assumes that we never get `None` or `""` items from CW, which is probably save to assume?

Reply:
The problematic case occurred for `min_mem_available_MB_chunks = [0.0]`. This should now be handled correctly.